<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MouseManager</class>
 <widget class="QDialog" name="MouseManager">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>295</width>
    <height>612</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MouseManager</string>
  </property>
  <layout class="QVBoxLayout">
   <item>
    <widget class="Q3GroupBox" name="LeftButton">
     <property name="title">
      <string>Left Button</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="LeftOperationLabel">
        <property name="text">
         <string>Operation:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="LeftOperationCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="MiddleButton">
     <property name="title">
      <string>Middle Button</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="MiddleOperationLabel">
        <property name="text">
         <string>Operation:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="MiddleOperationCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="Q3GroupBox" name="RightButton">
     <property name="title">
      <string>Right Button</string>
     </property>
     <layout class="QVBoxLayout">
      <item>
       <widget class="QLabel" name="RightOperationLabel">
        <property name="text">
         <string>Operation:</string>
        </property>
        <property name="wordWrap">
         <bool>false</bool>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QComboBox" name="RightOperationCombo"/>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="spacer2">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Expanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <pixmapfunction>qPixmapFromMimeSource</pixmapfunction>
 <customwidgets>
  <customwidget>
   <class>Q3GroupBox</class>
   <extends>QGroupBox</extends>
   <header>Qt3Support/Q3GroupBox</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
